packages/composition/dbaas/exoscale/postgres.yml

```diff
@@ -0,0 +1,148 @@
+                - fromConnectionSecretKey: POSTGRESQL_URL
+                  type: FromConnectionSecretKey
+                  name: POSTGRESQL_URL
+                - fromConnectionSecretKey: POSTGRESQL_DB
+                  type: FromConnectionSecretKey
+                  name: POSTGRESQL_DB
+                - fromConnectionSecretKey: POSTGRESQL_HOST
+                  type: FromConnectionSecretKey
+                  name: POSTGRESQL_HOST
+                - fromConnectionSecretKey: POSTGRESQL_PORT
+                  type: FromConnectionSecretKey
+                  name: POSTGRESQL_PORT
+                - fromConnectionSecretKey: ca.crt
+                  type: FromConnectionSecretKey
+                  name: ca.crt
+              patches:
+                - type: PatchSet
+                  patchSetName: annotations
+                - type: PatchSet
+                  patchSetName: labels
+                - type: FromCompositeFieldPath
+                  fromFieldPath: metadata.labels[crossplane.io/composite]
+                  toFieldPath: metadata.name
+                - type: ToCompositeFieldPath
+                  fromFieldPath: status.conditions
+                  toFieldPath: status.accessUserConditions
+                - type: FromCompositeFieldPath
+                  fromFieldPath: metadata.labels[crossplane.io/composite]
+                  toFieldPath: spec.writeConnectionSecretToRef.name
+                # Set postgres database version
+                - type: FromCompositeFieldPath
+                  fromFieldPath: spec..parameters.service.majorVersion
+                  toFieldPath: spec.forProvider.version
+                # Set postgres settings
+                - type: FromCompositeFieldPath
+                  fromFieldPath: spec.parameters.service.pgSettings
+                  toFieldPath: spec.forProvider.pgSettings
+                # Set zone
+                - type: FromCompositeFieldPath
+                  fromFieldPath: spec.parameters.service.zone
+                  toFieldPath: spec.forProvider.zone
+                # Set exoscale plan
+                - type: FromCompositeFieldPath
+                  fromFieldPath: spec.parameters.size.plan
+                  toFieldPath: spec.forProvider.size.plan
+                # Set maintenance day of week
+                - type: FromCompositeFieldPath
+                  fromFieldPath: spec.parameters.maintenance.dayOfWeek
+                  toFieldPath: spec.forProvider.maintenance.dayOfWeek
+                # Set maintenance time of day
+                - type: CombineFromComposite
+                  combine:
+                    variables:
+                      - fromFieldPath: spec.parameters.maintenance.hour
+                      - fromFieldPath: spec.parameters.maintenance.minute
+                    strategy: string
+                    string:
+                      fmt: "%s:%s:00"
+                  toFieldPath: spec.forProvider.maintenance.timeOfDay
```

Comment:
Why not keep the timeOfDay in the same format in the XRD? (hh:mm:ss). It doesn't make sense to translate [hour,minute] (XRD) -> [timeOfDay] (CRD) -> [hour,minute] (Exoscale API) . Specifying a time of day is more user-friendly than dedicated parameters for hour + minute.

Reply:
It was proposed like that in the AC. @Kidswiss do you see any issues if we keep the time format the same as from the managed resource?

Reply:
If it's easier go for @ccremer's suggestion.